kornia/geometry/boxes.py

```diff
@@ -217,33 +210,48 @@ def __init__(
         self._data = boxes
         self._mode = mode
 
-    def __getitem__(self, key) -> "Boxes":
+    def __getitem__(self, key) -> T:
         new_box = type(self)(self._data[key], False)
         new_box._mode = self._mode
         return new_box
 
-    def __setitem__(self, key, value: "Boxes") -> "Boxes":
+    def __setitem__(self, key, value: T) -> T:
         self._data[key] = value._data
         return self
 
+    def __mul__(self, other: T) -> T:
+        obj = self.clone()
+        obj._data = self.data * other.data
+        return obj
+
+    def __rmul__(self, other: Any) -> T:
+        obj = self.clone()
+        obj._data = self.data * other
+        return obj
+
+    def __add__(self, other: T) -> T:
+        obj = self.clone()
+        obj._data = self.data + other.data
+        return obj
+
     @property
     def shape(self):
         return self.data.shape
 
-    def get_boxes_shape(self) -> Tuple[torch.Tensor, torch.Tensor]:
+    def get_boxes_shape(self) -> Tuple[Tensor, Tensor]:
         r"""Compute boxes heights and widths.
 
         Returns:
             - Boxes heights, shape of :math:`(N,)` or :math:`(B,N)`.
             - Boxes widths, shape of :math:`(N,)` or :math:`(B,N)`.
 
         Example:
-            >>> boxes_xyxy = torch.tensor([[[1,1,2,2],[1,1,3,2]]])
+            >>> boxes_xyxy = Tensor([[[1,1,2,2],[1,1,3,2]]])
             >>> boxes = Boxes.from_tensor(boxes_xyxy)
             >>> boxes.get_boxes_shape()
             (tensor([[1., 1.]]), tensor([[1., 2.]]))
         """
-        boxes_xywh = cast(torch.Tensor, self.to_tensor("xywh", as_padded_sequence=True))
+        boxes_xywh = cast(Tensor, self.to_tensor("xywh", as_padded_sequence=True))
```

Comment:
should not need this cast here
```suggestion
        boxes_xywh = self.to_tensor("xywh", as_padded_sequence=True)
        if not isinstance(Tensor, boxes_xywh):
             raise
```